### PR TITLE
Fixed caps; added discovery endpoint

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/configure-client-management-openid-task.adoc [hunk 1]
```diff
@@ -120,7 +120,7 @@ If you have configured an OpenID Connect provider for client management, Anypoin
 [NOTE]
 The Refresh token can be selected only if the authorization grant type is also selected.
 
-In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc.
+In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc]:
```suggestion
In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`, where `$ISSUER` is the issuer configured as the provider. 
```

File: modules/ROOT/pages/configure-client-management-openid-task.adoc [hunk 2]
```diff
@@ -120,7 +120,7 @@ If you have configured an OpenID Connect provider for client management, Anypoin
 [NOTE]
 The Refresh token can be selected only if the authorization grant type is also selected.
 
-In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc.
+In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`, where `$ISSUER` is the issuer configured as the provider.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc]:
```suggestion
In addition to these scopes, if you configure the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform autopopulates its UI with all grant types supported by the provider. This includes grant types such as client credentials, password, and so on. To verify the grant types your provider supports, check their discovery endpoint using `__$ISSUER__/.well-known/openid-configuration`, where `_$ISSUER_` is the issuer configured as the provider. 
```

Comment (wwarner-salesforce) [inline modules/ROOT/pages/configure-client-management-openid-task.adoc]:
```suggestion
In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, and so on. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`, where `$ISSUER` is the issuer configured as the provider. 
```

**Review by wwarner-salesforce — APPROVED** [review]

**Review by kbostrom — APPROVED** [review]
Here you go!
(Going forward, we will probably boldface the field name....)

**Pull request merged**